src/views/tables/MarketsTable.tsx

```diff
@@ -263,7 +271,7 @@ export const MarketsTable = ({ className }: { className?: string }) => {
           navigate(`${AppRoute.Trade}/${market}`, { state: { from: AppRoute.Markets } })
         }
         defaultSortDescriptor={{
-          column: 'volume24H',
+          column: isTablet ? 'marketAndVolume' : 'volume24H',
```

Comment:
unfortunately we don't re-sort on screen size change, only when it's first initialized. no fix for now